yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1082,14 +1125,28 @@ func reportExclusionConstraintCreateTable(createTableNode *pg_query.Node_CreateS
 		//In case CREATE DDL has EXCLUDE USING gist(room_id '=', time_range WITH &&) - it will be included in columns but won't have columnDef as its a constraint
 		if column.GetColumnDef() == nil && column.GetConstraint() != nil {
 			if column.GetConstraint().Contype == pg_query.ConstrType_CONSTR_EXCLUSION {
+				colNames := getColumnNamesFromExclusions(column.GetConstraint().GetExclusions())
+				generatedConName := fmt.Sprintf("%s_%s_excl", tableName, strings.Join(colNames, "_"))
```

Comment:
This logic seems to be repeated everywhere. 
See if we can write a function `GenerateConstraintName(conType pg_query.ConstrType, tableName string, columns []string)`
that abstracts the prefix logic as well as the string concat?